react/features/virtual-background/functions.ts

```diff
@@ -18,6 +18,9 @@ export function checkBlurSupport() {
     return filterSupport;
 }
 
+export function checkVirtualBackgroundEnabled() : boolean {
+    return true ? APP.store.getState()['features/base/config'].enableVirtualBackground || APP.store.getState()['features/base/config'].enableVirtualBackground === undefined : false;
```

Comment:
This will break mobile. Why the true? Have you ran `npm run lint`, this will fail I think.

Reply:
I tested on android browser and app, now it's working.